src/components/KanaTable/KanaTable.tsx

```diff
@@ -0,0 +1,159 @@
+import styled from 'styled-components';
+import kanaType from '../../types/kana';
+import DataCell from './DataCell';
+
+type KanaTableProps = {
+  kana: kanaType[];
```

Comment:
Types are usually capitalized 
```suggestion
  kana: KanaType[];
```